scripts/release/config.js

```diff
@@ -2,6 +2,7 @@
 
 const dependencies = ['fbjs', 'object-assign', 'prop-types'];
 
+// TODO: enumerate all non-private package folders in packages/*?
```

Comment:
👍 I like this.

We could actually enumerate _all_ package folders (so we increment version numbers) but just only publish non-private ones.